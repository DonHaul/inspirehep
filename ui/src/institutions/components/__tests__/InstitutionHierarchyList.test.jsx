import React from 'react';
import { render } from '@testing-library/react';
import { fromJS } from 'immutable';
import InstitutionHierarchyList from '../InstitutionHierarchyList';

describe('InstitutionHierarchyList', () => {
  it('renders', () => {
    const hierarchies = fromJS([
      {
        name: 'Name1',
        acronym: 'NA',
      },
      {
        name: 'Name2',
      },
    ]);
    const { asFragment } = render(
      <InstitutionHierarchyList hierarchies={hierarchies} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
