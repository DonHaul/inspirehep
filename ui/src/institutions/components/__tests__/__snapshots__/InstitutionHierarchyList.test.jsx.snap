// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstitutionHierarchyList renders 1`] = `
<DocumentFragment>
  <div
    class="__InlineList__"
  >
    <ul>
      <li>
        <span>
          Name1
          <span>
             (NA)
          </span>
        </span>
         â€¢ 
      </li>
      <li>
        <span>
          Name2
        </span>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
